docs/custom_domains.rst

```diff
@@ -23,6 +23,10 @@ You can also host your documentation from your own domain.
    We don't currently support pointing subdomains or root domains to a project using A records.
    DNS A records require a static IP address and our IPs may change without notice.
 
+.. note::
+ 
+   If your custom domain was previously used in GitBook, contact GitBook support(via live chat in their website) to remove the domain name from their DNS Zone in order for your domain name to work with ReadtheDocs, else it will always redirect to GitBook.
```

Comment:
Maybe we can move this to the Troubleshooting section instead?

```suggestion
   If your custom domain was previously used in GitBook, contact GitBook support (via live chat in their website)
   to remove the domain name from their DNS Zone in order for your domain name to work with Read the Docs,
   else it will always redirect to GitBook.
```

Reply:
Sure, Lemme move it